workflow-templates/im-test-playwright-manual.yml

```diff
@@ -0,0 +1,224 @@
+# Workflow Code: SpunkyHamster_v1   DO NOT REMOVE
+# Purpose:
+#    Runs Playwright tests in Github Actions with a manual trigger.
+#    With the workflow the user specifies when they kick it off manually.
+#
+# Frequency:
+#    - This workflow is intended to be reusable, and used once per repository.
+#
+# Projects to use this Template with:
+#    - Testing projects that use Playwright.
+# # TODO: Prerequisites:
+#    - Add @estruyf/github-actions-reporter reporter to your playwright config when running in CI.  npm install --dev @estruyf/github-actions-reporter.
+#       - playwright.config.ts Example:  
+#         reporter: process.env.CI ? [
+#             ['list'],
+#             ['html'],
+#             ['@estruyf/github-actions-reporter'] 
+#           ] : [['list'],['html']], 
+
+name: 🧪 Run Playwright Tests
+run-name: 🧪 Run Playwright Tests on ${{ inputs.environment == 0 && 'dev' || inputs.environment }} with ${{ inputs.browser-project == 0 && 'chromium' || inputs.browser-project }} browser
+
+concurrency:
+  group: playwright-${{ github.event_name == 'workflow_dispatch' && github.run_id || github.ref  }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch-tag-sha:
+        description: Branch/Tag/SHA; The branch, tag or sha of the selenium tests should be run against.
+        required: false
+      environment:
+        description: The environment to test against
+        required: true
+        type: choice
+        options:
+          - dev
+          - qa
+          - stage
+          # - prod # Uncomment this line if you want to test against prod
+      browser-project:
+        description: Browser - what browser project to use. # Browser projects defined in the playwright.config.ts file.
+        type: choice
+        options:
+          - chromium
+          - firefox
+          - webkit
+          - all
```

Comment:
If the browser projects need to be defined in the playwright.config, does that mean the list of options here could change team by team? If so, we should add a TODO in here along the lines of `TODO: Add or remove any options that are or aren't in the playwright.config.ts for the playwright project`.

Reply:
Added todo.